main.go

```diff
@@ -1,11 +1,20 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"wedding_service/webserver"
+)
+
+var httpServer = &http.Server{Addr: fmt.Sprintf("localhost:%s", os.Getenv("WEDDING_SERVICE_HTTP_PORT"))}
+var httpsServer = &http.Server{Addr: fmt.Sprintf("localhost:%s", os.Getenv("WEDDING_SERVICE_HTTPS_PORT"))}
 
 func main() {
 	start()
 }
 
-func start() {
-	fmt.Println("coming soon")
+func start() error {
+	return webserver.Start(httpServer, httpsServer, filepath.Join("certificate", os.Getenv("LOCALHOST_CERT")), filepath.Join("certificate", os.Getenv("LOCALHOST_KEY")))
```

Comment:
we should probably make webserver into something, where we can initialize it better and with less parameters if any